submissions/AndriyM/oop_exercise_tiny_world/index.js

```diff
@@ -0,0 +1,43 @@
+const keys = ["species", "name", "gender", "saying", "legs"];
+class Inhabitant {
+	constructor(species, name, gender, legs, saying) {
+		this.species = species;
+		this.name = name;
+		this.legs = legs;
+		this.gender = gender;
+		this.saying = saying;
+	}
+	inhabitantShowProps() {
+		return keys.map((key) => this[key]).join("; ");
```

Comment:
`keys` and `key` are generic terms.
Any good reason to keep list of properties outside class definition?